diff --git a/lib/client.js b/lib/client.js
index a6834e160bfe9800347f05a028b7ed0ce4d929d7..31a74c95efbf6396245ccb0ca8fb3129ff5b54e4 100644
--- a/lib/client.js
+++ b/lib/client.js
@@ -2220,6 +2220,7 @@ window.__ModuleLoader__.load({
 			};
 			return (0, react_jsx_runtime.jsxs)("div", {
 				className: clsx(WorkspaceBrowser_module_css_default.root, !wide && WorkspaceBrowser_module_css_default.rail),
+				"data-dsh-workspace-drop-target": "",
 				children: [
 					(0, react_jsx_runtime.jsxs)("div", {
 						className: WorkspaceBrowser_module_css_default.sectionHeader,
